src/components/Search/SearchResultsListItem.tsx

```diff
@@ -125,7 +142,19 @@ function SearchResultsListItem({
                       className='time-icon'
                       fontSize='small'
                     />
-                    {String(dayjs(itemDate).format('MM/DD/YY HH:mm'))}
+
+                    {/* Tooltip with UTC time */}
+                    <Tooltip title={`${utcTime}`}>
```

Comment:
it would be good to extract this functionality to a separate react component, so that you can use it both here and in SelectedRevisionItem. Can you please do that?